site2/docs/administration-geo.md

```diff
@@ -45,6 +45,32 @@ All messages produced in any of the three clusters are delivered to all subscrip
 
 As stated in [Geo-replication and Pulsar properties](#geo-replication-and-pulsar-properties) section, geo-replication in Pulsar is managed at the [tenant](reference-terminology.md#tenant) level.
 
+The following example connects three clusters: **us-east**, **us-west**, and **us-cent**.
+
+### Connect Clusters
+
+To replicate among clusters, you must first configure each cluster to connect to the other. You can use the [`pulsar-admin`](reference-pulsar-admin.md#clusters) tool to create the connection.
+
+Run the following command in **us-west** to configure the connection to **us-east**:
+
+```shell
+$ bin/pulsar-admin clusters create \
+  --broker-url pulsar://<DNS-OF-US-EAST>:<PORT>	\
+  --url http://<DNS-OF-US-EAST>:<PORT> \
+  us-east
+```
+
+Then, run the following command in **us-west** to configure the connection to **us-cent**:
+
+```shell
+$ bin/pulsar-admin clusters create \
+  --broker-url pulsar://<DNS-OF-US-CENT>:<PORT>	\
+  --url http://<DNS-OF-US-CENT>:<PORT> \
+  us-cent
+```
+
+Analogous commands will need to be run in **us-east** and **us-cent** to create the remaining necessary cluster connections for replication. For clusters using a secure connection, see the [`pulsar-admin`](reference-pulsar-admin.md#clusters) create command. 
```

Comment:
```suggestion
### Connect replication clusters

To replicate data among clusters, you need to configure each cluster to connect to the other. You can use the [`pulsar-admin`](http://pulsar.apache.org/tools/pulsar-admin/) tool to create a connection.

**Example**

Suppose that you have 3 replication clusters: `us-west`, `us-cent`, and `us-east`.

1. Configure the connection from `us-west` to `us-east`. 
   
   Run the following command on `us-west`.

    ```shell
    bin/pulsar-admin clusters create \
    --broker-url pulsar://<DNS-OF-US-EAST>:<PORT> \
    --url http://<DNS-OF-US-EAST>:<PORT> \
    ```

    > #### Tip
    >
    > If you want to use a secure connection for a cluster, you can use the flag `--url-secure`. For more information, see [pulsar-admin clusters create](http://pulsar.apache.org/tools/pulsar-admin/).

2. Configure the connection from `us-west` to `us-cent`. 
   
   Run the following command on `us-west`.

    ```shell
    bin/pulsar-admin clusters create \
    --broker-url pulsar://<DNS-OF-US-CENT>:<PORT> \
    --url http://<DNS-OF-US-CENT>:<PORT> \
    ```

3. Run similar commands on `us-east` and `us-cent` to create connections among clusters.
```

Reply:
I added a commit that modified it in a similar way to your request. Please let me know if I need any further changes. I'm not sure about the usage of the link to pulsar admin: http://pulsar.apache.org/tools/pulsar-admin/. I used it because you suggested it, but it seems a bit unpolished. Here is a screenshot:

![Screen Shot 2021-03-11 at 10 15 49 PM](https://user-images.githubusercontent.com/47911938/110895621-57e23280-82b7-11eb-9a3b-84f644a22d4c.png)

Reply:
@michaeljmarshall thanks for your feedback.

1. `reference-pulsar-admin.md` is deprecated. Details: https://github.com/apache/pulsar/pull/9813
![image](https://user-images.githubusercontent.com/50226895/110899887-d46b2480-833c-11eb-989b-dcab95caf4c5.png)

2. It is better to use general links rather than specific links since here modifies master docs. Users can search and find their desired commands on http://pulsar.apache.org/tools/pulsar-admin/.

Reply:
Thank you for those references.